org.phoenicis.playonlinux.json

```diff
@@ -0,0 +1,80 @@
+{
+  "app-id": "org.phoenicis.playonlinux",
+  "runtime": "org.freedesktop.Platform",
+  "runtime-version": "18.08",
+  "sdk": "org.freedesktop.Sdk",
+  "sdk-extensions": [
+    "org.freedesktop.Sdk.Extension.openjdk11"
+  ],
+  "add-extensions": {
+    "org.freedesktop.Platform.Compat.i386": {
+      "directory": "lib/i386-linux-gnu",
+      "version": "18.08"
+    },
+    "org.freedesktop.Platform.Compat.i386.Debug": {
```

Comment:
Is the debug extension really needed? I see that Steam has it as well, hmm.

Reply:
Like you said: taken from Steam (many of our scripts install Steam games so it makes sense to me).